packages/slider/src/HandleController.ts

```diff
@@ -493,10 +493,8 @@ export class HandleController {
             [this.host.isLTR ? 'left' : 'right']: `${
                 model.normalizedValue * 100
             }%`,
+            right: 'auto',
             'z-index': zIndex.toString(),
-            // Allow setting background per-handle
```

Comment:
This shouldn't be removed, it should be updated to support the Core Tokens values.

Reply:
Now allowing background color and border color attributes in case of multiHandle
```
const style = {
            [this.host.isLTR ? 'left' : 'right']: `${
                model.normalizedValue * 100
            }%`,
            right: 'auto',
            'z-index': zIndex.toString(),
            ...(isMultiHandle && {
                'background-color': `var(--spectrum-slider-handle-background-color-${index}, var(--spectrum-slider-handle-background-color))`,
                'border-color': `var(--spectrum-slider-handle-border-color-${index}, var(--spectrum-slider-handle-border-color))`,
            }),
        };
 ```